test/e2e/config/packet-ci.yaml

```diff
@@ -54,7 +54,7 @@ providers:
   files:
   - sourcePath: "../../../metadata.yaml"
     targetName: "metadata.yaml"
-  - sourcePath: "../../../templates/cluster-template.yaml"
+  - sourcePath: "../../../templates/cluster-template-ci.yaml"
```

Comment:
was wrong from the previous PR that added the e2e, this is the correct template to use.